frontend/src/pages/admin-users.tsx

```diff
@@ -1,22 +1,88 @@
+import axiosInstance from '@/helpers/axios-instance';
+import { useEffect, useState } from 'react';
+import { toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+
+interface User {
+  _id: string;
+  fullName: string;
+  role: 'ADMIN' | 'USER';
+  email: string;
```

Comment:
This guy shouldn't he use the enum defined below? 

Can you kindly replace all the interface to types